packages/core/src/test/awsService/accessanalyzer/iamPolicyChecks.test.ts

```diff
@@ -0,0 +1,793 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from 'assert'
+import * as sinon from 'sinon'
+import {
+    _readCustomChecksFile,
+    getResultCssColor,
+    IamPolicyChecksWebview,
+    PolicyChecksError,
+    renderIamPolicyChecks,
+} from '../../../awsService/accessanalyzer/vue/iamPolicyChecks'
+import { FakeExtensionContext } from '../../../test/fakeExtensionContext'
+import fs from 'fs'
```

Comment:
use our [fs.ts](https://github.com/aws/aws-toolkit-vscode/blob/4bb2fc1395d98ac71d90d83360c8f578ed0eccc2/packages/core/src/shared/fs/fs.ts) module wherever possible. Avoid using node APIs directly. Our fs.ts module should remove the need for that in most fs-related cases.